client/views/incidentReports/incidentTable.jade

```diff
@@ -9,26 +9,20 @@ template(name="incidentTable")
         th Start Date
         th End Date
         th.locations-header Locations
-        th Status
```

Comment:
Is it okay to remove these columns from the /extract-incidents page? Perhaps we should create a separate incident table for the curator inbox.